modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -0,0 +1,147 @@
+/singleton/reagent/inaprovaline
+	description = "Inaprovaline is a chemical that narrows blood vessels, raises blood pressure, and helps the blood pump more efficiently. Commonly used to stabilize patients."
```

Comment:
"Inaprovaline is a chemical that narrows blood vessels, raises blood pressure, and helps blood pump more efficiently. Commonly used to stabilize patients.  It acts as a cardiostimulant, which stabilises myocardial contractility, working towards maintaining a steady pulse and blood pressure. Inaprovaline also acts as a weak analgesic"